### DISCUSSION TITLE
Does Kupo understand time relative to slot no. on different networks? Does it make sense to return it?

### DISCUSSION THREAD

**Discussion by ross-spencer** [CardanoSolutions/kupo#178] [Q&A]
If I understand correctly, time can be approximated off-chain for a given slot if we understand the network parameters and boundaries relative to a Unix timestamp. Does Kupo have this information behind the scenes? We have slot in the last-checkpoint (`X-Most-Recent-Checkpoint`) in the header, an approx. timestamp could also be helpful to avoid going out to another third party library (that might not exist for a language). Could it be added if it's there?

**Comment by KtorZ** [comment]
It could totally be added indeed.
